<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="Background" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget #Background{
background-color:rgb(102, 84, 94)}
</string>
   </property>
   <widget class="QPushButton" name="add_button">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>500</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
background-color: rgb(246, 224, 181);
font: 75 18pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Add Deadline</string>
    </property>
   </widget>
   <widget class="QLabel" name="form_name">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>320</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 25pt &quot;MS Shell Dlg 2&quot;; 
color: rgb(246, 224, 181);
text-align: right;</string>
    </property>
    <property name="text">
     <string>ADD NEW DEADLINE</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="name_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;MS Shell Dlg 2&quot;; 
color: rgb(246, 224, 181);
text-align: right;</string>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_field">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>100</y>
      <width>350</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
font-size: 20px;
color: black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="date_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;MS Shell Dlg 2&quot;; 
color: rgb(246, 224, 181);
text-align: right;</string>
    </property>
    <property name="text">
     <string>Date:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="description_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;MS Shell Dlg 2&quot;; 
color: rgb(246, 224, 181);
text-align: right;</string>
    </property>
    <property name="text">
     <string>Description:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="description_field">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>320</y>
      <width>350</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
font-size: 20px;
color: black;</string>
    </property>
   </widget>
   <widget class="QLabel" name="date_field">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>170</y>
      <width>191</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;MS Shell Dlg 2&quot;; 
color: rgb(246, 224, 181);
text-align: right;</string>
    </property>
    <property name="text">
     <string>MM/DD/YYYY</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="assigned_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>151</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;MS Shell Dlg 2&quot;; 
color: rgb(246, 224, 181);
text-align: right;</string>
    </property>
    <property name="text">
     <string>Assigned To:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="assigned_field">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>230</y>
      <width>350</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
font-size: 20px;
color: black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
